src/app/rpmostree-override-builtins.cxx

```diff
@@ -89,6 +91,47 @@ handle_override (RPMOSTreeSysroot  *sysroot_proxy,
   g_autoptr(GVariant) options = g_variant_ref_sink (g_variant_dict_end (&dict));
 
   g_autoptr(GVariant) previous_deployment = rpmostree_os_dup_default_deployment (os_proxy);
+//  g_autofree char *transaction_address = NULL;                                          
+//  if (!rpmostree_os_call_refresh_md_sync(os_proxy,&transaction_address,cancellable,error))
+//    return FALSE;
```

Comment:
Then here you would call `rpmostree_transaction_get_response_sync`. See `rpmostree_builtin_refresh_md`.

Reply:
But we only want to do this if using the `Search` API.